app/models/upload.rb

```diff
@@ -0,0 +1,9 @@
+class Upload < ActiveRecord::Base
+  belongs_to :user
+  has_many :likes
+  has_many :comments
+
+  has_attached_file :image, :styles => { :medium => "500x500>", :thumb => "200x200>" },
+                    :default_url => ActionController::Base.helpers.asset_path('missing.jpeg')
+  validates_attachment_content_type :image, :content_type => /\Aimage\/.*\Z/
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)<br>Style/RegexpLiteral: Use `%r` around regular expression. (https://github.com/bbatsov/ruby-style-guide#percent-r)